### DISCUSSION TITLE
persistent copy fails

### DISCUSSION THREAD

**Discussion by EelcoHoff** [fabianishere/udm-iptv#245] [Q&A]
Hi there,

The command: 

     cp /etc/udm-iptv.conf /mnt/persistent/udm-iptv.conf

 fails wtih:

      ........:~# cp /etc/udm-iptv.conf /mnt/persistent/udm-iptv.conf cp: cannot create regular file '/mnt/persistent/udm-iptv.conf': No such
      file or directory

Wat am i doing wrong?

kind regards,
Eelco

**Comment by fabianishere** [comment]
This won't work anymore with the new UDM firmware. I have removed the advice from the README.
